Update operation-status-endpoints.md to add bulk/{bulkUuid}/operation-status/{status} description (#6087)
magento/devdocs: src/guides/v2.3/rest/operation-status-endpoints.md

}
```

## Get operations count by bulk uuid and status.

> The `GET <host>/rest/<store_code>/V1/bulk/:bulkUuid/status/:status` endpoint returns amount of operations from the bulk batch in particular status.

Reviewer: Please, make sure that you are mentioning the correct endpoint here. It looks like it supposed to be `/V1/bulk/:bulkUuid/operation-status/:status` but I might be wrong. 

Thank you.

}
```

## Get operations count by bulk uuid and status.

> The `GET <host>/rest/<store_code>/V1/bulk/:bulkUuid/operation-status/:status` endpoint returns amount of operations from the bulk batch in particular status.

Reviewer 2: ```suggestion
The `GET <host>/rest/<store_code>/V1/bulk/:bulkUuid/operation-status/:status` endpoint returns the number of operations from the bulk batch that have the specified status.
```

> 

Reviewer 2: Add a table that lists the possible status values. You can use the same text as above, except make it a simple table with 5 lines, rather than forcing line breakes with `<br/> elements.

Author: done
